templates/lookup/ifupdown__dynamic_packages.j2

```diff
@@ -0,0 +1,21 @@
+{% set ifupdown__tpl_packages = [] %}
+{% if ifupdown__combined_interfaces|d() %}
+{%   for interface, params in ifupdown__combined_interfaces.iteritems() %}
+{%     if (params.state|d('present') == 'present' and params.type|d()) %}
+{%       if params.type == 'bridge' %}
+{%         set _ = ifupdown__tpl_packages.append('bridge-utils') %}
+{%       elif params.type == 'vlan' %}
+{%         set _ = ifupdown__tpl_packages.append('vlan') %}
+{%       elif params.type == 'bond' %}
+{%         if ansible_distribution_release == 'wheezy' %}
+{%           set _ = ifupdown__tpl_packages.append('ifenslave-2.6') %}
+{%         else %}
+{%           set _ = ifupdown__tpl_packages.append('ifenslave') %}
+{%         endif %}
+{%       endif %}
+{%     endif %}
+{%   endfor %}
+{% endif %}
+{% if ifupdown__tpl_packages %}
```

Comment:
What happens if `ifupdown__tpl_packages` remains empty :wink: I guess in that case the role would fail. Maybe remove the condition in this case. The YAML just be returned either way.